googlemediaframework/src/main/java/com/google/android/libraries/mediaframework/exoplayerextensions/RendererBuilderFactory.java

```diff
@@ -19,34 +19,42 @@
 import android.content.Context;
 import android.net.Uri;
 
+import com.google.android.exoplayer.drm.MediaDrmCallback;
+import com.google.android.exoplayer.extractor.mp4.Mp4Extractor;
+
 /**
  * Generate a renderer builder appropriate for rendering a video.
  */
 public class RendererBuilderFactory {
 
-  /**
-   * Create a renderer builder which can build the given video.
-   * @param ctx The context (ex {@link android.app.Activity} in whicb the video has been created.
-   * @param video The video which will be played.
-   */
-  public static ExoplayerWrapper.RendererBuilder createRendererBuilder(Context ctx,
-                                                                       Video video) {
-    switch (video.getVideoType()) {
-      case HLS:
-        return new HlsRendererBuilder(ExoplayerUtil.getUserAgent(ctx),
-                                      video.getUrl(),
-                                      video.getContentId());
-      case DASH:
-        return new DashRendererBuilder(ExoplayerUtil.getUserAgent(ctx),
-                                       video.getUrl(),
-                                       video.getContentId(),
-                                       new WidevineTestMediaDrmCallback(video.getContentId()),
-                                       null); // TODO: Pass in DebugTextView here.
-      case MP4:
-        // TODO: DebugTextView.
-        return new DefaultRendererBuilder(ctx, Uri.parse(video.getUrl()), null);
-      default:
-        return null;
+    /**
+     * Create a renderer builder which can build the given video.
+     * @param ctx The context (ex {@link android.app.Activity} in whicb the video has been created.
+     * @param video The video which will be played.
+     */
+    public static ExoplayerWrapper.RendererBuilder createRendererBuilder(Context ctx,
+                                                                         Video video) {
+        String userAgent = ExoplayerUtil.getUserAgent(ctx);
+
+        switch (video.getVideoType()) {
+            case HLS:
+                return new HlsRendererBuilder(ctx,
+                        userAgent,
```

Comment:
Code style here is off, indents should be +4 spaces if continuing a line (see style guide)